astro/src/content/blog/hundreds-millions-entities.mdx

```diff
@@ -0,0 +1,240 @@
+        at org.apache.ibatis.executor.SimpleExecutor.prepareStatement(SimpleExecutor.java:89)
+        at org.apache.ibatis.executor.SimpleExecutor.doUpdate(SimpleExecutor.java:49)
+        at org.apache.ibatis.executor.BaseExecutor.update(BaseExecutor.java:117)
+        at org.apache.ibatis.executor.CachingExecutor.update(CachingExecutor.java:76)
+        at org.apache.ibatis.session.defaults.DefaultSqlSession.update(DefaultSqlSession.java:197)
+        ... 15 common frames omitted 
+Caused by: org.postgresql.util.PSQLException: Connection to DBHOSTNAME:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.
+        at org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:346)
+        at org.postgresql.core.ConnectionFactory.openConnection(ConnectionFactory.java:54)
+        at org.postgresql.jdbc.PgConnection.<init>(PgConnection.java:273)
+        at org.postgresql.Driver.makeConnection(Driver.java:446)
+        at org.postgresql.Driver.connect(Driver.java:298)
+        at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:138)
+        at com.zaxxer.hikari.pool.PoolBase.newConnection(PoolBase.java:359)
+        at com.zaxxer.hikari.pool.PoolBase.newPoolEntry(PoolBase.java:201)
+        at com.zaxxer.hikari.pool.HikariPool.createPoolEntry(HikariPool.java:470)
+        at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:733)
+        at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:712)
+        at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
+        ... 3 common frames omitted 
+Caused by: java.net.ConnectException: Connection refused 
+        at java.base/sun.nio.ch.Net.pollConnect(Native Method) 
+        at java.base/sun.nio.ch.Net.pollConnectNow(Net.java:672)
+        at java.base/sun.nio.ch.NioSocketImpl.timedFinishConnect(NioSocketImpl.java:554)
+        at java.base/sun.nio.ch.NioSocketImpl.connect(NioSocketImpl.java:602)
+        at java.base/java.net.SocksSocketImpl.connect(SocksSocketImpl.java:327)
+        at java.base/java.net.Socket.connect(Socket.java:633)
+        at org.postgresql.core.PGStream.createSocket(PGStream.java:243)
+        at org.postgresql.core.PGStream.<init>(PGStream.java:98)
+        at org.postgresql.core.v3.ConnectionFactoryImpl.tryConnect(ConnectionFactoryImpl.java:136)
+        at org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:262)
+        ... 14 common frames omitted 
+```
+
+The database disk filling up caused a hard failure of FusionAuth. We determined the disk was full by reviewing our monitoring systems. If this had been a customer deployment, we would have been alerted well prior to failure and taken steps to resolve it. Since this was a testing instance, the alerts were silenced.
```

Comment:
I think this paragraph should be an aside (note) or called out in some other manner.  I would hate for a reader skimming and seeing the issues not realizing it is because this is a test instance.  Are maybe moving the note to the head of this section so they are crystal clear.

Reply:
Thanks @mark-robustelli , great point.